src/main/scala/com/microsoft/azure/cosmosdb/spark/AuthConfig.scala

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
java and scala license headers should start with /* not /**

/** is used for javadoc, you may get invalid doc taging.